test/integration/provider/admin/onboarding/wizard/request_controller_test.rb

```diff
@@ -31,11 +31,6 @@ class Provider::Admin::Onboarding::Wizard::RequestControllerTest < ActionDispatc
 
     put provider_admin_onboarding_wizard_request_path(request: { path: 'some invalid !!! path' })
     assert_response :success
-
-    ProxyDeploymentService.any_instance.expects(call: false)
-    put provider_admin_onboarding_wizard_request_path(request: { path: '/path' })
-    assert_response :success
-    assert_match 'please try again', response.body
```

Comment:
This doesn't look as a V1 specific test. Could you explain why test failed?

Reply:
I have changed the test_api method in https://github.com/3scale/porta/blob/2542542cc655c9aa3e79c734b1391949394aebc4/app/forms/onboarding/request_form.rb. which is called in the update method of request_controller so the test was failing

Reply:
Agree with @akostadinov , I don't think we have to remove this.